### DISCUSSION TITLE
BT-6110 Contract EU Funds Details

### DISCUSSION THREAD

**Discussion by kletersteegp** [OP-TED/eForms-SDK#437] [Q&A] [answered]
Is there a reason why BT-6110 is not a field in de notices 25 tm 40?
The CELEX spreadsheet states that this field is optional in these notice. It is also in a field in the fields.json.
But is not defined in for example 29.json from the notice-types. And also not a field in the other json files for the notices.

<img width="967" alt="Knipsel" src="https://user-images.githubusercontent.com/111063502/235666274-74cd38f1-5290-43c1-a729-640316ffeb34.PNG">

**Comment by YvesJo** [comment, marked as answer]
Not all fields resulting from the Regulation amendment could be added to the notice type definitions yet and we are doing our best to reduce this list of missing fields as we go.
